app/controllers/menu_days_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class MenuDaysController < ApplicationController
+  def index
+    @menu_days = MenuDays.find_by_day(1).dishes
+  end
+
+  def select_day
+    @menu_days = MenuDays.find_by_day(params[:day]).dishes
```

Comment:
Look. You have variable in plural days. so logically in the end you should get list of menu_days, but you don't get. It should describe what the variable means. @menu_day_dishes or @dishes_for_menu_day